### DISCUSSION TITLE
currentday highlighting (?)

### DISCUSSION THREAD

**Discussion by Symbiot78** [totaldebug/atomic-calendar-revive#1475] [Support]
Is it possible to highlight the current day?
I see old posts referencing this:

currentDay {
   color: yellow !important;
   background: green !important;
 }

but I haven't been able to make that work.. tried different iterations but to no avail. 
Has this been deprecated at some point?

tried .single-event-container.currentDay but nope.
looked at the dev tools in browser but can't see anything referencing currentday.. there was something for the date but nothing for the whole line.
I'd like to highlight the whole line and be able to set a color of the text as well.

**Comment by realtek2017** [comment]
You actually need:

```
card_mod:
  style: |
    .current-day  { 
    color: orange !important;
    }
```
